lib/control.mli

```diff
@@ -20,6 +20,9 @@ val interrupt : bool ref
 (** Coq interruption: set the following boolean reference to interrupt Coq
     (it eventually raises [Break], simulating a Ctrl-C) *)
 
+val break : bool ref
```

Comment:
Shouldn't this be in a different file?

Reply:
It relies on receiving a signal, just like `interrupt`, so this seemed a reasonable place to put it.  It could go into `DebugHook` but not into `Tactic_debug` since that's plugin code.